src/utils/vote-presets.js

```diff
@@ -3,8 +3,40 @@ import { erc20ABI } from 'wagmi'
 import BigNumber from 'bignumber.js'
 
 import VaultABI from './abis/Vault.json'
+import EthPNTABI from './abis/EthPNT.json'
 import settings from '../settings'
 
+const prepareInflationProposal = async (receiverAddress, rawAmount) => {
```

Comment:
The name rawAmount makes me think the *10^18 is already applied

Reply:
I agree

Reply:
Should I switch amount and rawAmount?